lua/core/midi.lua

```diff
@@ -194,6 +226,7 @@ local to_data = {
 }
 
 --- convert msg to data (midi bytes)
+-- @tparam msg
```

Comment:
maybe type info too?

Reply:
"type info"?  I'm not sure whats needed. Do you mean the data type of `msg`?

Reply:
sorry, yeah.  that's exactly what i meant.